tests/docker/docker-compose.yml

```diff
@@ -6,9 +6,34 @@ networks:
     driver: bridge
 
 services:
+  minio:
```

Comment:
does this commit that enables minio need to be sequenced before the tests that use s3 in ducktape?